### PR TITLE
Release v4.3.2

### PR DESCRIPTION
**Pull request by tscheypidi**
Please fill following information
(Add additional info if you think its important and not covered by this Pull Request (PR)):

## Purpose of this PR

Release of MAgPIE v4.3.2

### CODE REVIEW

**Review by abhimishr — DISMISSED** [review]
🚀 Good to go 💎 🙌🏽 📈

File: modules/35_natveg/static/declarations.gms [hunk 1]
```diff
@@ -9,19 +9,21 @@ positive variables
   v35_secdforest(j,ac) 		Detailed stock of secdforest (mio. ha)
   v35_other(j,ac)      		Detailed stock of other land (mio. ha)
   vm_landdiff_natveg        Aggregated difference in other land compared to previous timestep (mio. ha)
-  vm_hvarea_secdforest(j,ac)                    Harvested area from secondary forest (mio. ha)
-  vm_hvarea_other(j,ac)                         Harvested area from other land (mio. ha)
-  vm_hvarea_primforest(j)                       Harvested area from primary forest (mio. ha)
+  v35_hvarea_secdforest(j,ac)                    Harvested area from secondary forest (mio. ha)
+  v35_hvarea_other(j,ac)                         Harvested area from other land (mio. ha)
+  v35_hvarea_primforest(j)                       Harvested area from primary forest (mio. ha)
+  vm_cost_hvarea_natveg(i)						blub
```
Comment (FelicitasBeier) [inline modules/35_natveg/static/declarations.gms:15]: Please add a variable description here

File: modules/39_landconversion/global_static_aug18/not_used.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# |  (C) 2008-2021 Potsdam Institute for Climate Impact Research (PIK)
+# |  authors, and contributors see CITATION.cff file. This file is part
+# |  of MAgPIE and licensed under AGPL-3.0-or-later. Under Section 7 of
+# |  AGPL-3.0, you are granted additional permissions described in the
+# |  MAgPIE License Exception, version 1.0 (see LICENSE file).
+# |  Contact: magpie@pik-potsdam.de
+name,type,reason
```
Comment (FelicitasBeier) [inline modules/39_landconversion/global_static_aug18/not_used.txt:7]: empty file. I think this doesn't have to be included, right?

File: modules/50_nr_soil_budget/exoeff_aug16/declarations.gms [hunk 3]
```diff
@@ -1,4 +1,4 @@
-*** |  (C) 2008-2020 Potsdam Institute for Climate Impact Research (PIK)
+*** |  (C) 2008-2021 Potsdam Institute for Climate Impact Research (PIK)
 *** |  authors, and contributors see CITATION.cff file. This file is part
```
Comment (FelicitasBeier) [inline modules/50_nr_soil_budget/exoeff_aug16/declarations.gms:2]: Just wondering: Does this already include Benni's latest bugfix?

File: modules/52_carbon/normal_dec17/preloop.gms [hunk 4]
```diff
@@ -1,20 +1,11 @@
-*** |  (C) 2008-2020 Potsdam Institute for Climate Impact Research (PIK)
+*** |  (C) 2008-2021 Potsdam Institute for Climate Impact Research (PIK)
```
Comment (FelicitasBeier) [inline modules/52_carbon/normal_dec17/preloop.gms:1]: This file is now empty. I think should be deleted

**Review by FelicitasBeier — CHANGES_REQUESTED** [review]
Minor code cleanup changes requested.

**Comment by abhimishr** [comment]
@tscheypidi what would be the best way to address the requested changes? Branch out from my commits at https://github.com/magpiemodel/magpie/pull/249, call it rc in my fork, make a PR for my fork's rc with main model's rc?

File: modules/10_land/landmatrix_dec18/start.gms [hunk 5]
```diff
@@ -0,0 +1,17 @@
+*** |  (C) 2008-2021 Potsdam Institute for Climate Impact Research (PIK)
+*** |  authors, and contributors see CITATION.cff file. This file is part
+*** |  of MAgPIE and licensed under AGPL-3.0-or-later. Under Section 7 of
+*** |  AGPL-3.0, you are granted additional permissions described in the
+*** |  MAgPIE License Exception, version 1.0 (see LICENSE file).
+*** |  Contact: magpie@pik-potsdam.de
+
+*due to some rounding errors the input data currently may contain in some cases
```
[the remark this answers is not in the document]
Comment (abhimishr) [reply modules/10_land/landmatrix_dec18/start.gms:8]: True. Maybe we keep it for next data iteration of data preparation?

File: modules/15_food/anthropometrics_jan18/intersolve.gms [hunk 6]
```diff
@@ -18,14 +18,23 @@ option nlp = conopt4
...
 
-p15_iteration_counter(t)= p15_iteration_counter(t) + 1;
-
+* A new iteration is started
+p15_iteration_counter(t) = p15_iteration_counter(t) + 1;
+* The set current iter includes only one element with the set element 
+* of the current iteration, e.g. "iter2"
+curr_iter15(iter15) = no;
+curr_iter15(iter15)$(ord(iter15)=p15_iteration_counter(t)) = yes;
+* Now we also define a set for the last iteration
```
[the remark this answers is not in the document]
Comment (abhimishr) [reply modules/15_food/anthropometrics_jan18/intersolve.gms:33]: @bodirsky please check

File: modules/15_food/anthropometrics_jan18/presolve.gms [hunk 7]
```diff
@@ -218,7 +218,15 @@ vm_dem_food.fx(i,knf)=0;
 
 *** Food demand model is calculated the first time for the current time step, using standard prices
 
-p15_iteration_counter(t)    =0;
+* activating the first iteration
```
[the remark this answers is not in the document]
Comment (abhimishr) [reply modules/15_food/anthropometrics_jan18/presolve.gms:221]: @bodirsky please check

File: modules/50_nr_soil_budget/exoeff_aug16/declarations.gms (hunk 3 above)
Comment (abhimishr) [reply modules/50_nr_soil_budget/exoeff_aug16/declarations.gms:2]: Not sure. Can you check @bodirsky ? Seems to be related to https://github.com/magpiemodel/magpie/pull/244 which was already part of https://github.com/magpiemodel/magpie/pull/249 so yes

File: modules/52_carbon/normal_dec17/preloop.gms (hunk 4 above)
Comment (abhimishr) [reply modules/52_carbon/normal_dec17/preloop.gms]: Fixed via 65bd3d4

File: modules/39_landconversion/global_static_aug18/not_used.txt (hunk 2 above)
Comment (abhimishr) [reply modules/39_landconversion/global_static_aug18/not_used.txt]: Fixed via 65bd3d4

File: modules/35_natveg/static/declarations.gms (hunk 1 above)
Comment (abhimishr) [reply modules/35_natveg/static/declarations.gms]: Fixed via 65bd3d4

File: modules/50_nr_soil_budget/exoeff_aug16/declarations.gms (hunk 3 above)
Comment (abhimishr) [reply modules/50_nr_soil_budget/exoeff_aug16/declarations.gms:2]: If this is another bugfix which came after https://github.com/magpiemodel/magpie/pull/249 then it won't be part of this planned release

File: modules/15_food/anthropometrics_jan18/presolve.gms (hunk 7 above)
[the remark this answers is not in the document]
Comment (flohump) [reply modules/15_food/anthropometrics_jan18/presolve.gms:221]: Yes. This is needed for the food demand model.

File: modules/15_food/anthropometrics_jan18/intersolve.gms (hunk 6 above)
[the remark this answers is not in the document]
Comment (bodirsky) [reply modules/15_food/anthropometrics_jan18/intersolve.gms:33]: "previous" seems indeed better wording to me!
How do i change this in the RC?

File: modules/50_nr_soil_budget/exoeff_aug16/declarations.gms (hunk 3 above)
Comment (bodirsky) [reply modules/50_nr_soil_budget/exoeff_aug16/declarations.gms:2]: Which bugfix do you refer to?

File: modules/15_food/anthropometrics_jan18/intersolve.gms (hunk 6 above)
[the remark this answers is not in the document]
Comment (abhimishr) [reply modules/15_food/anthropometrics_jan18/intersolve.gms:33]: i can pass this edit as a commit as I already have a cloned rc locally

[the remark this answers is not in the document]
Comment (abhimishr) [reply modules/15_food/anthropometrics_jan18/intersolve.gms]: Fixed with https://github.com/magpiemodel/magpie/pull/262/commits/14228c4cb3aff3313251ae2e1122ee8bbbdcc104

**Comment by abhimishr** [comment]
I marked most of the requests resolved. @FelicitasBeier can you also check if https://github.com/magpiemodel/magpie/pull/262#discussion_r596869827 is okay? (Also if you'd like to re-open the changes I marked resolved)

**Review by FelicitasBeier — APPROVED** [review]
seems fine

**Review by abhimishr — APPROVED** [review]
🚀 ready to go

**Pull request merged**